docker/Dockerfile.pyt_build

```diff
@@ -0,0 +1,71 @@
+ARG BASE_IMAGE=pytorch/pytorch:2.6.0-cuda12.6-cudnn9-devel
+ARG DEVEL_IMAGE=devel
+
+FROM ${BASE_IMAGE} AS devel
+WORKDIR /workspace/deps
+
+RUN ls -la /usr/local/cuda && echo "/usr/local/cuda exists" || echo "/usr/local/cuda does NOT exist"
+
+# Set CUDA_HOME and update PATH and LD_LIBRARY_PATH
+ENV CUDA_HOME=/usr/local/cuda-12.6
```

Comment:
how about ENV CUDA_HOME=/usr/local/cuda
if I was not wrong,  /usr/local/cuda -> /usr/local/cuda-<ver>  in the ngc img